addons/survey/views/survey_templates.xml

```diff
@@ -206,6 +225,7 @@
     <!-- Question widgets -->
     <template id="question_container" name="Survey: question container">
         <t t-set="answer_lines" t-value="answer.user_input_line_ids.filtered(lambda line: line.question_id == question)"/>
+        <t t-set="useKeySelection" t-value="len(question.suggested_answer_ids) &lt; 26 and survey.questions_layout == 'page_per_question'"/>
```

Comment:
Could set that variable as a `data-use-key-selection` of the question wrapper to avoid having to re-check that 26 length thing on the JS side.